db/migrate/20231025193639_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Let's stick to the ERD diagram and ensure that instead of `user_id`, the column is named `author_id`. Here's a way to do this: 👇 
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```
![image](https://github.com/2004-okari/BLOG_APP/assets/46839548/13947d58-cc56-428f-aec4-f227c922d56e)